app/src/main/java/org/simple/clinic/summary/bloodpressures/BloodPressureSummaryViewInit.kt

```diff
@@ -0,0 +1,13 @@
+package org.simple.clinic.summary.bloodpressures
```

Comment:
Was this class left behind by mistake? It still refers `NewBloodPressureSummaryViewEffect`, which I think was renamed.

Reply:
I forgot I only changed the Kotlin file name rather than the `sealed class` name. Fixed it